test/parallel/test-assert-deep.js

```diff
@@ -1026,7 +1033,15 @@ test('Strict equal with identical objects that are not identical ' +
     {
       code: 'ERR_ASSERTION',
       name: 'AssertionError',
-      message: /\.\.\./g
+      message: 'Expected values to be strictly deep-equal:\n' +
+        '+ actual - expected\n' +
+        '\n' +
+        '  {\n' +
+        '    symbol0: Symbol(),\n' +
+        '    symbol10: Symbol(),\n' +
+        '    symbol11: Symbol(),\n' +
+        '    symbol12: Symbol(),\n' +
+        '    symbol13: Symbol(),\n'
```

Comment:
The former output seems more useful to the reader, no? It is clear that the values where expected to be strictly deep-equal. They are not but the entries are not reference-equal, which was formerly reported.
The cut-off also seems difficult to follow vs. `...` from the former output.

Reply:
TBH, I like the newer output better, it tells me better (and in a more concise way) the differences!  
The only thing I might agree upon is that the message `Values have same structure but are not reference-equal:` is better than `Expected values to be strictly deep-equal:`, but I am also not sure about it 😄